app/uk/gov/hmrc/ctcguaranteebalancerouter/models/errors/PresentationError.scala

```diff
@@ -31,22 +30,15 @@ object PresentationError {
   def badRequestError(message: String): PresentationError =
     StandardError(message, ErrorCode.BadRequest)
 
+  def invalidGuaranteeTypeError(message: String): PresentationError =
+    StandardError(message, ErrorCode.InvalidGuaranteeType)
+
   def forbiddenError(message: String): PresentationError =
     StandardError(message, ErrorCode.Forbidden)
 
   def notFoundError(message: String): PresentationError =
     StandardError(message, ErrorCode.NotFound)
 
-  def rateLimited(message: String): PresentationError =
```

Comment:
Assume we no longer require this since now  synchronous?

Reply:
It was just marked as unused -- I suspect it was copied from elsewhere, never used, and not cleaned up.